### PR TITLE
assertion improvement on cluster DRS

### PR DESCRIPTION
**Pull request by DaanHoogland**
### Description

This PR ...

Fixes: #8507 

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI
- [x] test (unit or integration

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [x] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test alma8 vmware-70u3

**Review by vishesh92 — APPROVED** [review]
clgtm

File: test/integration/smoke/test_cluster_drs.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
 import time
```
Comment (DaanHoogland) [inline test/integration/smoke/test_cluster_drs.py:24]:
```suggestion
from collections.abc import Iterable

```

**Comment by DaanHoogland** [comment]
@blueorangutan test alma8 vmware-70u3

**Comment by shwstppr** [comment]
@DaanHoogland should this target to 4.19 branch?

**Comment by DaanHoogland** [comment]
@shwstppr I can rebase if needed, but didn´t think it relevant or 4.19 (might be mistaking)

**Comment by vishesh92** [comment]
@DaanHoogland Test is in 4.19 as well. So, we need to fix this for 4.19 as well.

**Comment by DaanHoogland** [comment]
@blueorangutan test alma8 vmware-70u3

**Comment by shwstppr** [comment]
@blueorangutan test alma8 vmware-70u3

File: test/integration/smoke/test_cluster_drs.py [hunk 2]
```diff
@@ -256,7 +294,8 @@ def test_02_balanced_drs_algorithm(self):
             migration["virtualmachineid"]: migration["destinationhostid"] for migration in migrations
         }
 
-        self.assertEqual(len(vm_to_dest_host_map), 1, msg="DRS plan should have 1 migrations")
+        # this is one if no svm is considered to be migrated, it might be higher
+        self.assertTrue(len(vm_to_dest_host_map) <= 1, msg="DRS plan should have 1 migrations")
```
Comment (DaanHoogland) [inline test/integration/smoke/test_cluster_drs.py:298]:
```suggestion
        self.assertTrue(len(vm_to_dest_host_map) >= 1, msg="DRS plan should have at least 1 migrations")
```

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Comment by DaanHoogland** [comment]
@blueorangutan test alma9 vmware-70u3

**Comment by vishesh92** [comment]
@blueorangutan test alma9 vmware-70u3 keepEnv

**Comment by vishesh92** [comment]
@blueorangutan package

**Comment by vishesh92** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test alma9 vmware-70u3

**Comment by sureshanaparti** [comment]
@blueorangutan package

**Comment by vishesh92** [comment]
@blueorangutan test alma9 vmware-70u3

**Comment by vishesh92** [comment]
@blueorangutan test alma9 vmware-70u3

**Review by sureshanaparti — APPROVED** [review]
clgtm

**Comment by DaanHoogland** [comment]
At least the cluster DRS test errors are gone now. What do we think @vishesh92 @sureshanaparti ?

**Comment by vishesh92** [comment]
> At least the cluster DRS test errors are gone now. What do we think @vishesh92 @sureshanaparti ?

Yeah. Let's merge it.

**Review by vishesh92 — APPROVED** [review]

**Comment by sureshanaparti** [comment]
No issues with cluster DRS test here, merging this based on review / test result.

**Pull request merged**